game_eggs/among_us/bettercrewlink_server/egg-better-crewlink-server.json

```diff
@@ -0,0 +1,92 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-07-25T21:57:48+02:00",
+    "name": "BetterCrewlink Server",
+    "author": "tobiornotto@gmail.com",
+    "description": "An egg designed to allow support for Proximity Chat in Among Us using BetterCrewLink Server",
+    "features": null,
+    "docker_images": {
+        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-12": "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-12"
```

Comment:
You should use the new yolks images: `ghcr.io/parkervcp/yolks:nodejs_12`

Reply:
Updated and tested!